tool/jt.rb

```diff
@@ -517,6 +518,26 @@ def dis(file)
     puts Pathname(file).sub_ext('.ll')
   end
 
+  def environment
+    puts "Environment"
+    env_vars = ['JAVA_HOME','RUBY_BIN','GRAALVM_BIN','GRAAL_HOME',
+    'JVMCI_BIN','JVMCI_GRAAL_HOME','GRAAL_JS_JAR',
+    'SL_JAR','OPENSSL_PREFIX','AOT_BIN','TRUFFLERUBY_CEXT_ENABLED',
+    'TRUFFLERUBYOPT','RUBYOPT']
```

Comment:
I would like to have `PATH` in there, it's often the source of problems.

Reply:
Also `TRUFFLERUBY_RESILIENT_GEM_HOME`

Reply:
BTW, `%w[foo bar]` can be nicely used here to avoid quotes and commas.

Reply:
Fixed all these.